01-CSS_Transform/assignment/css/_variables.scss

```diff
@@ -0,0 +1,28 @@
+//colors
+$color1: black;
+$color2: white;
+$color3: #71c7ec;
```

Comment:
One thing that's really helpful is to name variables something semantic. `color3` doesn't tell me what it is. Instead maybe use `light-blue` or something like that.